docs/operations/controller-configmap.md

```diff
@@ -15,11 +15,54 @@ metadata:
   name: numaflow-controller-config
 data:
   controller-config.yaml: |
+    defaults:
+      containerResources: |
+        ...
     isbsvc:
       jetstream:
         ...
 ```
 
+### Default Controller Configuration
+
+Currently, we support configuring the init and main container resources for steps across all the pipelines. The configuration is under `defaults` key in the ConfigMap.
+
+For example, to set the default container resources for steps across all the pipelines:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: numaflow-controller-config
+data:
+  controller-config.yaml: |
+    defaults:
+      containerResources: |
+        limits:
+          memory: "256Mi"
+          cpu: "200m"
+        requests:
+          memory: "128Mi"
+          cpu: "100m"
+```
+
+Similarly, you can set the default container limits resources for steps across all the pipelines:
```

Comment:
This section has no difference from above, can be removed?

Reply:
it has a difference, it explains not setting request. let me know if need to remove this

Reply:
Why not setting `requests` it will default to limits?

Reply:
As the ResourceRequirements are directly sourced to k8s API that we use ([refer](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2553)).

Reply:
removed it!